unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Clipboard/Implementation/IClipboardImplementation.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+
+internal interface IClipboardImplementation
```

Comment:
Its hard to me to see the word `Implementation` in an interface. The `handler` name maybe its more appropiate because it's what we were using for similar interfaces?
```suggestion
internal interface IClipboardHandler
```

Reply:
I will rename it, but these are more platform dependent implementations than handlers